package.json

```diff
@@ -39,67 +39,30 @@
   },
   "author": "kiwi.com",
   "license": "MIT",
+  "resolutions": {
+    "@types/react": "^17.0.26",
+    "@types/react-dom": "^17.0.9"
+  },
```

Comment:
Multiple versions of `@types/react` and `@types/react-dom` caused problems similar to the ones we had with `@types/react-native`, i.e. a bunch of errors inside `@types/react` and `@types/react-dom` complaining that modules were redeclared with different types.